src/applications/disability-benefits/686c-674/config/chapters/add-a-child/child-place-of-birth/child-place-of-birth.js

```diff
@@ -74,39 +74,34 @@ export const uiSchema = {
         },
         city: {
           'ui:title': 'City or county',
+          'ui:required': formData =>
+            isChapterFieldRequired(formData, 'addChild'),
         },
       },
       childStatus: {
         'ui:title': "Your child's status (check all that apply)",
-        biological: {
-          'ui:title': 'Biological',
-        },
-        adopted: {
-          'ui:title': 'Adopted',
-        },
-        notCapable: {
-          'ui:title': 'Not capable of self-support',
-        },
-        stepchild: {
-          'ui:title': 'Stepchild',
-        },
-        dateBecameDependent: merge(
-          currentOrPastDateUI('Date stepchild became dependent'),
-          {
-            'ui:options': {
-              expandUnder: 'stepchild',
-              expandUnderCondition: true,
-              keepInPageOnReview: true,
-            },
-          },
-        ),
+        'ui:widget': 'radio',
+        'ui:required': formData => isChapterFieldRequired(formData, 'addChild'),
```

Comment:
Just a heads up, changing this to an enum and using radio actually only allows a user to select one. I think we need to keep it how it was originally, possibly use a checkbox widget.

Reply:
Changed this back to a list of checkboxes, I didn't realize that someone needed to be able to pick more than one